<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Card</class>
 <widget class="QDialog" name="Card">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QLineEdit" name="line_edit_number">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_name">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_date">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_cvv">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_num">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Card number</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cardholder name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_date">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date expire</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_cvv">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CVV</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="button_ok">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_clear">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_close">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_card_number">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1234 1235 1235 1235</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_card_valid_thru">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>65</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>VALID
THRU</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_card_date">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>12/20</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_card_name">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NSimSun</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PODUSENKO VIKTOR</string>
   </property>
  </widget>
  <zorder>label_card_name</zorder>
  <zorder>label_card_date</zorder>
  <zorder>label_card_number</zorder>
  <zorder>label_card_valid_thru</zorder>
  <zorder>line_edit_number</zorder>
  <zorder>line_edit_name</zorder>
  <zorder>line_edit_date</zorder>
  <zorder>line_edit_cvv</zorder>
  <zorder>label_num</zorder>
  <zorder>label_name</zorder>
  <zorder>label_date</zorder>
  <zorder>label_cvv</zorder>
  <zorder>button_ok</zorder>
  <zorder>button_clear</zorder>
  <zorder>button_close</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
